<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Физиологические исследования a-analyser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QWidget" name="wgtClient" native="true">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="acDeviceControl">
   <property name="text">
    <string>Управление оборудованием</string>
   </property>
   <property name="toolTip">
    <string>Управление оборудованием...</string>
   </property>
  </action>
  <action name="acDataBaseProperty">
   <property name="text">
    <string>Свойства...</string>
   </property>
   <property name="toolTip">
    <string>Свойства базы данных</string>
   </property>
  </action>
  <action name="acDataBaseExport">
   <property name="text">
    <string>Экспорт...</string>
   </property>
   <property name="toolTip">
    <string>Экспорт базы данных</string>
   </property>
  </action>
  <action name="acDataBaseImport">
   <property name="text">
    <string>Импорт...</string>
   </property>
   <property name="toolTip">
    <string>Импорт базы данных</string>
   </property>
  </action>
  <action name="acDataBaseSelect">
   <property name="text">
    <string>Выбрать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>acDeviceControl</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDeviceControl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acDataBaseProperty</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDataBaseProperty()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acDataBaseExport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDataBaseExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acDataBaseImport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDataBaseImport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acDataBaseSelect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDataBaseSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDeviceControl()</slot>
  <slot>onDataBaseProperty()</slot>
  <slot>onDataBaseExport()</slot>
  <slot>onDataBaseImport()</slot>
  <slot>onDataBaseSelect()</slot>
 </slots>
</ui>
